packages/ansible-language-server/src/utils/execPath.ts

```diff
@@ -11,8 +13,22 @@ import { ExtensionSettings } from "../interfaces/extensionSettings";
 export function getAnsibleCommandExecPath(
   name: string,
   settings: ExtensionSettings,
+  context: WorkspaceFolderContext,
 ): string {
-  return name === "ansible-lint"
-    ? settings.validation.lint.path
-    : path.join(path.dirname(settings.ansible.path), name);
+  let pathFromSettings =
+    name === "ansible-lint"
+      ? settings.validation.lint.path
+      : path.join(path.dirname(settings.ansible.path), name);
+  return replaceWorkspaceFolderInPath(pathFromSettings, context);
```

Comment:
I am sure the vscode api has generic support for replacing all variables in the settings, please use that instead of trying to do it DIY.

Reply:
I have been unable to find it.
This suggests it doesn't exist: https://github.com/microsoft/vscode/issues/46471

Reply:
However if you can find it I'd be happy to use it!